### PR TITLE
cleanup: add region tags to .sh and .yaml files

### PR DESCRIPTION
**Pull request by ckim328**
Added region tags to k8 yaml files and .sh files! 

~For shell files, I decided to have the license placed before the region tags and `#!/bin/bash` 
Let me know if I should revert~

I also noticed that there were many more yaml files than what `grep -roh "kind:" . | wc -w ` showed. 
Should I add region tags to those yaml files as well?

### CODE REVIEW

File: anthos-baremetal-edge-deployment/scripts/create-primary-gsa.sh
Comment (NimJay) [inline anthos-baremetal-edge-deployment/scripts/create-primary-gsa.sh:78]: Question: Is the removal of this chunk of `.sh` scripts intentional?

File: anthos-baremetal-edge-deployment/scripts/delete-abm-gsa-keys-gcp.sh
Comment (NimJay) [inline anthos-baremetal-edge-deployment/scripts/delete-abm-gsa-keys-gcp.sh:17]: Thought: 

@ckim328, thanks for pointing out that you:
> decided to have the license placed before the region tags and #!/bin/bash

Judging from [this old blog post](https://ajurasz.github.io/2016-11-02-watch-were-you-put-your-license-headers/) from 2016, it's possible to run into issues with Docker when the shebang `#!/bin/bash` comes _after_ the license header.
The issue the author faced: `/bin/sh` was used instead of `/bin/bash`.
I'm happy to leave things as is — but I'll let the bash experts @askmeegs and @Shabirmean have the final say.

**Review by NimJay — CHANGES_REQUESTED** [review]
The region tags are named well.
Thanks for being very careful with the changes, @ckim328!

I've left 2 comments.
We'll definitely wait for a review from @Shabirmean and @askmeegs on everything.

File: anthos-baremetal-edge-deployment/scripts/create-primary-gsa.sh
Comment (ckim328) [reply anthos-baremetal-edge-deployment/scripts/create-primary-gsa.sh:78]: omgosh i def removed this by accident great catch 👏  ill add it back in!!

File: anthos-baremetal-edge-deployment/scripts/delete-abm-gsa-keys-gcp.sh
Comment (ckim328) [reply anthos-baremetal-edge-deployment/scripts/delete-abm-gsa-keys-gcp.sh:17]: I'll be editing those! thanks Nim!!

File: anthos-baremetal-edge-deployment/acm-config-sink/namespaces/pos/api-server-lb.yaml
Comment (Shabirmean) [inline anthos-baremetal-edge-deployment/acm-config-sink/namespaces/pos/api-server-lb.yaml:16]: Sorry if I missed any previous conversations. But is this region tag formed following some convention/guidance? This looks a bit too long.

Comment (ckim328) [reply anthos-baremetal-edge-deployment/acm-config-sink/namespaces/pos/api-server-lb.yaml:16]: For our sample app repos, we have this format:
```
REGION TAG FORMAT: YAML  (with metadata name)
<product-prefix>_<directory-with-underscores>_<filename-without-extension>_<kind>_<metadata.name>
REGION TAG FORMAT: YAML  (w/out metadata name)
<product-prefix>_<parent-directory-with-underscores>_<directory-with-underscores>_<filename-without-extension>_<kind>
REGION TAG FORMAT: SHELL
<product-prefix>_<directory-with-underscores>_<filename-without-extension>
```
It's to make our lives easier (I'm not about to go through every yaml file 😅 )

I know some of these region tags are non-descriptive, but with our sample app repos these are just to get it into sample browser (it's not as high visibility as your efforts with nodejs-container repo :) )

that being said, the approach I took with boa and microservices-demo was to edit their `make-release.sh` ([example](https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/6bc21926ad2eb48d7a591a9e0ebd5ca81732d5e0/release/make-release.sh#L55)), and add region tags to the release yaml files. If that's the approach we can take in this repo, I can follow up with you!

Comment (Shabirmean) [reply anthos-baremetal-edge-deployment/acm-config-sink/namespaces/pos/api-server-lb.yaml:16]: Thank you for the detailed explanation. I understand it now. For sure we might want to have a similar process to what we see in the other samples app once we have this formally launched.

File: anthos-baremetal-edge-deployment/acm-config-sink/system/repo.yaml
Comment (askmeegs) [inline anthos-baremetal-edge-deployment/acm-config-sink/system/repo.yaml:16]: 😆

File: anthos-baremetal-edge-deployment/roles/abm-software/files/external-secrets-crd.yaml
Comment (askmeegs) [inline anthos-baremetal-edge-deployment/roles/abm-software/files/external-secrets-crd.yaml:16]: we might want to very quickly check that there's no character length limit on region tags.

**Review by askmeegs — APPROVED** [review]

File: anthos-baremetal-edge-deployment/roles/abm-software/files/external-secrets-crd.yaml
Comment (ckim328) [reply anthos-baremetal-edge-deployment/roles/abm-software/files/external-secrets-crd.yaml:16]: 104 characters 👍

**Review by Shabirmean — APPROVED** [review]
LGTM!

Thank you very much for doing this @ckim328. I think the surface you have covered here suffices. We don't necessarily have to cover the other files inside the `anthos-baremetal-edge-deployment` folder.

**Pull request merged**
